src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -170,6 +172,9 @@ private static String injectRealAccounts(String jsonString) {
                 .replace("${test.student1}", TestProperties.inst().TEST_STUDENT1_ACCOUNT)
                 .replace("${test.student2}", TestProperties.inst().TEST_STUDENT2_ACCOUNT)
                 .replace("${test.instructor}", TestProperties.inst().TEST_INSTRUCTOR_ACCOUNT)
+                .replace("${test.unreg}", TestProperties.inst().TEST_UNREG_ACCOUNT)
+                .replace("${test.admin}", TestProperties.inst().TEST_ADMIN_ACCOUNT)
+                .replace("${test.truncatedAdmin}", StringHelper.truncate(TestProperties.inst().TEST_ADMIN_ACCOUNT, Const.SystemParams.USER_ID_MAX_DISPLAY_LENGTH))
```

Comment:
Might not make sense to include ${test.truncatedAdmin} since truncation of the admin's account happens only for display, and this method is about the values that get inserted into the database.